packages/modelence/src/app/index.ts

```diff
@@ -184,6 +184,14 @@ const localConfigMap = {
   GOOGLE_AUTH_ENABLED: '_system.user.auth.google.enabled',
   GOOGLE_AUTH_CLIENT_ID: '_system.user.auth.google.clientId',
   GOOGLE_AUTH_CLIENT_SECRET: '_system.user.auth.google.clientSecret',
+  RESEND_API_KEY: '_system.email.resend.apiKey',
+  SES_REGION: '_system.email.ses.region',
+  SES_ACCESS_KEY_ID: '_system.email.ses.accessKeyId',
+  SES_SECRET_ACCESS_KEY: '_system.email.ses.secretAccessKey',
+  SMTP_HOST: '_system.email.smtp.host',
+  SMTP_PORT: '_system.email.smtp.port',
+  SMTP_USER: '_system.email.smtp.user',
+  SMTP_PASS: '_system.email.smtp.pass',
```

Comment:
Can we prefix these configs (at least the new ones) with `MODELENCE_`? And specifically `MODELENCE_EMAIL` to match the full config key? For example `MODELENCE_EMAIL_SMTP_HOST`.
Otherwise I'm worried this will collide with other environment variables that were not meant for Modelence.